### PR TITLE
Some changes for Requiem.

### PR DESCRIPTION
**Pull request by MichaelDietzel**
Added loadorder information for thetrader's "Requiem 1.9.3 Patch Central"

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -10426,6 +10426,667 @@ plugins:
...
+      - 'Requiem - Bugsmasher Edition.esp'
+      - 'Requiem - Legendary Bugsmasher Edition.esp'
+      - 'Requiem - No Messages.esp'
+      - 'Requiem - No Overencumbered Messages.esp'
+      - 'Requiem - Restored Rewarding Sounds.esp'
+      - 'Requiem - Restored Saving Messages.esp'
+      - 'Requiem - Vanilla Shrines.esp'
+      - 'Requiem - Hearthfire.esp'
+
+
```
Comment (WrinklyNinja) [inline masterlist.yaml]: There's not much point making another commit to correct only this, but in future please don't leave trailing whitespace. All good editors have an option to trim it for you automatically.

**Comment by WrinklyNinja** [comment]
I can't help but suspect that most of the load after entries you've added are unnecessary: for example, does `Requiem - Improved Closefaced Helmets.esp` really conflict with and need to load after 47 other patches, or were there just 47 mods listed before it in whatever load order you're referencing?

**Comment by MichaelDietzel** [comment]
Hi,

the last three entries with the long lists are specially listed by the author of these patches to be loaded last. And since I
don't know, which of these compatibility patches a user uses I added all of them in the load after list. And I didn't want to go
with a simple priority value.

Regards
Mimos / Michael

Am 12.12.2015 um 12:38 schrieb Oliver Hamlet:

**Comment by MichaelDietzel** [comment]
Update: well, I guess you are right and some could be removed and he just listed them in that order for not makin it too
complicated for humans to read. I'll take a look into it.

Hi,

the last three entries with the long lists are specially listed by the author of these patches to be loaded last. And since I
don't know, which of these compatibility patches a user uses I added all of them in the load after list. And I didn't want to go
with a simple priority value.

Regards
Mimos / Michael

Am 12.12.2015 um 12:38 schrieb Oliver Hamlet:

**Comment by Freso** [comment]
I'm closing this 1½ year old PR in favour of https://github.com/loot/skyrim/pull/136 – if you feel that PR is missing something, feel free to comment on it or make a follow-up PR. Thanks for your work on this!

**Pull request closed without merging**